src/main/java/com/checkout/ApiClientImpl.java

```diff
@@ -26,17 +26,17 @@
 import static com.checkout.ClientOperation.PUT;
 import static com.checkout.common.CheckoutUtils.validateParams;
 
-class ApiClientImpl implements ApiClient {
+public class ApiClientImpl implements ApiClient {
```

Comment:
Does it need to be public?

Reply:
Yes, otherwise we are not able to use in `com.checkout.four.CheckoutApiImpl` due the package protection